packages/calypso-e2e/src/lib/blocks/cover-block.ts

```diff
@@ -0,0 +1,41 @@
+import { ImageBlock } from './image-block';
+
+const coverStylesArray = [ 'Default', 'Bottom Wave', 'Top Wave' ] as const;
+export type coverStyles = typeof coverStylesArray[ number ];
+
+/**
+ * Represents the Cover block.
+ */
+export class CoverBlock extends ImageBlock {
```

Comment:
Question: is it possible to do what you want to do in CoverBlock without using inheritance?

Since we moved off the inheritance train sometime back in August, I feel that things have been much easier to work with.

Reply:
It is possible, although it will entail copy-pasting some methods as the Cover block is technically an Image block with extra features. We can also extract those methods to a separate helper, but I'm not sure if that would be less confusing than the class extension. What do you think?

Reply:
TBH, I don't think there will be other situations where we could merge blocks like that. With TS it's also easier to follow the inheritance I guess? It should be OK to leave this one as it makes sure e.g. the image upload will be handled consistently for both blocks at all times.

Reply:
Let's leave it as is for now!